source/configuration/read.txt

```diff
@@ -80,58 +80,46 @@ You can configure the following properties to read from MongoDB:
 
        The connector provides the following partitioners:
 
-       - ``MongoDefaultPartitioner``
-            **Default**. Wraps the MongoSamplePartitioner and provides
-            help for users of older versions of MongoDB.
-
-       - ``MongoSamplePartitioner``
+       - ``SamplePartitioner``
              **Requires MongoDB 3.2**. A general purpose partitioner for
              all deployments. Uses the average document size and random
              sampling of the collection to determine suitable
              partitions for the collection. For configuration 
-             settings for the MongoSamplePartitioner, see
-             :ref:`conf-mongosamplepartitioner`.
+             settings for the SamplePartitioner, see
+             :ref:`conf-samplepartitioner`.
 
-       - ``MongoShardedPartitioner``
+       - ``ShardedPartitioner``
              A partitioner for sharded clusters. Partitions the
              collection based on the data chunks. Requires read access
              to the ``config`` database. For configuration settings for
-             the MongoShardedPartitioner, see
-             :ref:`conf-mongoshardedpartitioner`.
-
-       - ``MongoSplitVectorPartitioner``
-             A partitioner for standalone or replica sets. Uses the
-             :dbcommand:`splitVector` command on the standalone or the
-             primary to determine the partitions of the database.
-             Requires privileges to run :dbcommand:`splitVector`
-             command. For configuration settings for the
-             MongoSplitVectorPartitioner, see
-             :ref:`conf-mongosplitvectorpartitioner`.
-
-       - ``MongoPaginateByCountPartitioner``
-             A slow, general purpose partitioner for all deployments.
-             Creates a specific number of partitions. Requires a query
-             for every partition. For configuration settings for the
-             MongoPaginateByCountPartitioner, see
-             :ref:`conf-mongopaginatebycountpartitioner`.
+             the ShardedPartitioner, see :ref:`conf-shardedpartitioner`.
 
-       - ``MongoPaginateBySizePartitioner``
+       - ``PaginateBySizePartitioner``
              A slow, general purpose partitioner for all deployments.
              Creates partitions based on data size. Requires a query
              for every partition. For configuration settings for the
-             MongoPaginateBySizePartitioner, see
-             :ref:`conf-mongopaginatebysizepartitioner`.
+             PaginateBySizePartitioner, see 
+             :ref:`conf-paginatebysizepartitioner`.
+
+       - ``PaginateIntoPartitionsPartitioner``
+             A partitioner that creates a specified number of 
+             partitions for a collection. Uses the count and the 
```

Comment:
Avoid mixing a high level description "average document size" with a function name (e.g. "the count [of a collection]") as objects in a conjunction. Instead, consider replacing "the count" with "the total number of documents".